templates/default/marathon.yaml.erb

```diff
@@ -0,0 +1,8 @@
+# Generated by Chef, local modifications will be overwritten
+
+<%= JSON.parse(({ 'instances' => @instances }).to_json).to_yaml %>
+
+init_config:
```

Comment:
That would be a bit better, but would argue that keeping it the same as the Mesos configuration file is more important for consistency. Unless we refactor that as well as part of this PR?